src/test/java/com/marklogic/kafka/connect/sink/AbstractIntegrationSinkTest.java

```diff
@@ -81,4 +100,8 @@ protected final void retryIfNotSuccessful(Runnable r, long sleepTime, int attemp
             }
         }
     }
+
+    public Map<String, Object> getTaskConfig() {
```

Comment:
This is fine to have - I was thinking that since `taskConfig` is protected, a subclass could just directly access it. I do that a lot in tests, though much less often in application code. I think you can make `taskConfig` private as a result, and this can also be protected so that there's no implication that anything besides a subclass would use this. But it's fine the way it is too.